searchapp/migrations/0013_auto_20190812_1532.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.1 on 2019-08-12 10:02
```

Comment:
I'm not sure if we should be pushing migrations. I'm not entirely sure about this.